pkg/agent/flowexporter/ipfix/ipfix_process.go

```diff
@@ -28,17 +29,14 @@ var _ IPFIXExportingProcess = new(ipfixExportingProcess)
 // IPFIXExportingProcess interface is added to facilitate unit testing without involving the code from go-ipfix library.
 type IPFIXExportingProcess interface {
 	LoadRegistries()
```

Comment:
As this function in go-ipfix is part of registry package, defining separate interface will be helpful and will aid in loading it once.